js/ast.go

```diff
@@ -320,21 +328,28 @@ func (s *Scope) Unscope() {
 
 ////////////////////////////////////////////////////////////////
 
+// INode is an interface for AST nodes
+type INode interface {
+	String() string
+	Raw() string
```

Comment:
I don't think `Raw` is a very accurate name. Why not call it `JS` or `Format` or something similar? I like `JS()` because it implies from any node we can print its valid JS.

Reply:
I believe parsers (or lexers maybe?) generally use `String` to convert it back to actual JavaScript, but since that's already taken I ended up naming it `Raw` instead. What's most important is the word we think of when we convert a parsed format to a string, "Raw" came to mind and is probably easier to find in any docs. It's also somewhat obvious that the result will be a string.

Reply:
Sure, `String` returns a string representation of the struct, this doesn't have to be valid JavaScript. I don't see what Raw means here, a raw representation of the struct? What does that mean? I think renaming it to `JS` or `ToJS` is much clearer.

Reply:
I'm okay with renaming it to `JS`.